mscore/palette.cpp

```diff
@@ -730,12 +782,69 @@ void Palette::leaveEvent(QEvent*)
       {
       if (currentIdx != -1) {
             QRect r = idxRect(currentIdx);
-            currentIdx = -1;
+            PaletteBox* pb = mscore->getPaletteBox();
+            if (!filterActive || !pb->getKeyboardNavigation())
+                  currentIdx = -1;
             update(r);
             }
       }
 
 //---------------------------------------------------------
+//   nextPaletteElement
+//---------------------------------------------------------
+void Palette::nextPaletteElement()
+      {
+      if (!filterActive)
```

Comment:
Advanced workspace in French. Type "ba". You should see that the tremolo palette doesn't change but every element matches the filter but "filterActive" is false and so it's not possible to go though the palette.